jekyll/_cci2/workflows.md

```diff
@@ -37,6 +37,8 @@ Refer to the [Workflows]({{ site.baseurl }}/2.0/faq) section of the Migration FA
 
 _For a full specification of the_ `workflows` _key, see the [Workflows]({{ site.baseurl }}/2.0/configuration-reference/#workflows) section of the Configuration Reference document._
 
+**Note:** Projects configured with Workflows often include multiple jobs that share similar or identical syntax for Docker images, environment variables, or particular `run` steps. Refer the [YAML Anchors/Aliases](http://yaml.org/spec/1.2/spec.html#id2765878) documentation for information about how to alias and reuse syntax to keep your `.circleci/config.yml` file small. See the [Reuse YAML in the CircleCI Config](https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/) blog post for a summary.
```

Comment:
Suggested change for first sentence:

"Projects configured with Workflows often include multiple jobs that share syntax for Docker images, environment variables, or `run` steps."

- "share" implies "similar or identical"
- "particular" should be applied to all three things that are shared, or to none of them.